src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/adjustments/AdjudicationsService.kt

```diff
@@ -40,6 +53,55 @@ class AdjudicationsService(
   override suspend fun retryCreateMapping(message: String) {}
 }
 
+internal fun ReportedAdjudicationResponseV2.toNomisAdjudication() = CreateAdjudicationRequest(
+  adjudicationNumber = reportedAdjudication.adjudicationNumber + 1_000_000, // So we can test basic skeleton in T3 a cheeky hack to ensure number is way out of range of the sequence
+  incident = IncidentToCreate(
+    reportingStaffUsername = reportedAdjudication.createdByUserId,
+    incidentDate = reportedAdjudication.incidentDetails.dateTimeOfIncident.toLocalDate(),
+    incidentTime = reportedAdjudication.incidentDetails.dateTimeOfIncident.toLocalTime().toString(),
+    reportedDate = reportedAdjudication.incidentDetails.dateTimeOfDiscovery.toLocalDate(),
+    reportedTime = reportedAdjudication.incidentDetails.dateTimeOfDiscovery.toLocalTime().toString(),
+    internalLocationId = reportedAdjudication.incidentDetails.locationId,
+    details = reportedAdjudication.incidentStatement.statement,
+    prisonId = reportedAdjudication.originatingAgencyId,
+    prisonerVictimsOffenderNumbers = reportedAdjudication.offenceDetails.victimPrisonersNumber?.let { listOf(it) } ?: emptyList(),
```

Comment:
I think maybe you could use [listOfNotNull](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/list-of-not-null.html) here.

Reply:
That sounds good. Not seen that before